package.json

```diff
@@ -14,25 +14,30 @@
     "url": "https://github.com/uber/react-map-gl.git"
   },
   "license": "MIT",
-  "main": "dist/index.js",
-  "module": "dist-es6/index.js",
+  "main": "dist/es5/index.js",
+  "module": "dist/esm/index.js",
+  "esnext": "dist/es6/index.js",
   "files": [
     "src",
-    "dist",
-    "dist-es6"
+    "dist"
   ],
   "scripts": {
     "start": "(cd examples/main && (path-exists node_modules || npm i) && npm run start-local)",
-    "build": "npm run clean && npm run build-es5 && npm run build-es6",
-    "build-es5": "mkdir -p dist && babel src --out-dir dist --plugins=transform-es2015-modules-commonjs,transform-runtime --source-maps inline",
-    "build-es6": "mkdir -p dist-es6 && babel src --out-dir dist-es6 --source-maps inline",
-    "clean": "rm -fr dist/* dist-es6/*",
+    "clean": "rm -fr dist dist-es6 && mkdir -p dist/es5 dist/esm dist/es6",
+    "build": "npm run clean && npm run build-es6 && npm run build-esm && npm run build-es5 && npm run build-size",
```

Comment:
These scripts are getting out of control. Can we add a bash build script instead so that these do not get published?

Reply:
Yes... I think we are soon going to want to create a helper module with build scripts, to simplify upgrading all modules.